test/napi/src/js/export.rs

```diff
@@ -27,7 +27,7 @@ fn simple_add(a: f64, b: f64) -> f64 {
 }
 
 #[neon::export(name = "renamedAdd")]
-fn renamed_add(a: f64, b: f64) -> f64 {
+fn rs_renamed_add(a: f64, b: f64) -> f64 {
```

Comment:
I added `rs_` prefixes here because the tests were useless after this change (the override matched the default).